library/mageia

```diff
@@ -2,11 +2,11 @@ Maintainers: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
 GitRepo: https://github.com/juanluisbaptiste/docker-brew-mageia.git
 
 Tags: 6, latest
-GitCommit: 6230a3fc5dd30e5419a391f4613eb3ca5d81f46a
+GitCommit: e690ea85472945459e1401c4dc6b7568f11d1e2c
 GitFetch: refs/heads/dist
 Directory: 6
 
 Tags: 5,5.1
-GitCommit: 6230a3fc5dd30e5419a391f4613eb3ca5d81f46a
+GitCommit: e690ea85472945459e1401c4dc6b7568f11d1e2c
 GitFetch: refs/heads/dist
 Directory: 5
```

Comment:
The `Dockerfile` in this directory is referencing `rootfs.tar.xz`, but the directory contains `rootfs.tar.gz` -- was that an intentional mixup? :innocent:

Reply:
Fixed.